### DISCUSSION TITLE
How to label content for different needs?

### DISCUSSION THREAD

**Discussion by syeda-git** [squidfunk/mkdocs-material#7765] [Q&A]
Hello,
We have two products with 80% of content same. We want to serve one doc site for both the products. I am thinking of putting labels to say at the top of the topics or at specific content/para that informs users that this topic/content belongs to product A or B or both. When the users search, the preview should show the product categorization.

Is tags a good way to go or is there a better way to handle this use case?
https://squidfunk.github.io/mkdocs-material/setup/setting-up-tags/#setting-up-tags

Thanks...
Syed

**Comment by kamilkrzyskow** [comment]
Hello @syeda-git,
yes, tags can be used for pages and will take priority in the search, so it's a good use case for that. They will be displayed at the top of the page. However, it's not optimized for tagging specific sections of the page, so if you need a more specific approach you'd have to find a plugin or create a custom solution.
